app_an01009/src/core/xua_conf.h

```diff
@@ -15,10 +15,9 @@
  * Note, we check if they are already defined in CMakeLists.txt
  */
 
-/*** Defines relating to basic functionality ***/
+#define XUA_BMAX_POWER (50) /* 100 mA */
```

Comment:
As mentioned, the whole board can draw just over 1 watt so it might result in the device being kicked off the bus?  250 mA would be safer with a comment to justify it